TADashboard/src/main/java/page_objects/DataProfilesPage.java

```diff
@@ -1,23 +1,25 @@
 package page_objects;
 
 import com.logigear.control.common.imp.CheckBox;
+import com.logigear.control.common.imp.Label;
 import com.logigear.control.common.imp.Link;
 import utils.common.Constants;
 
 public class DataProfilesPage extends BasePage {
-    private CheckBox dynamicCheckbox = new CheckBox("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']/preceding-sibling::td/input");
-    private Link dynamicDataProfile = new Link("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']");
-    private Link dynamicPresetAction = new Link("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']/following-sibling::td[@class='center']/a/parent::td");
+    private CheckBox dynamicDataProfileCheckbox = new CheckBox("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']/preceding-sibling::td/input");
+    private Label dynamicDataProfile = new Label("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']");
+    private Link dynamicDataProfileLink = new Link("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']/a");
```

Comment:
why do you need one element for `td` then another one for `a`?

Reply:
Because at td tag when I use method click(), it's not work. I am so confuse about it.